plugin/semantic_highlighting.py

```diff
@@ -97,6 +80,10 @@ def _render_with_plain_string_stackframes(
                 backtrace += '\n'
             backtrace += f'<div>{nums}{ctx}</div>'
 
+        semantic_tokens_html = ''
+        for info in semantic_info:
+            semantic_tokens_html += f'<div>Type: {info[0]}, Modifiers: {info[1]} <a>{info[2]}</a></div>'
```

Comment:
I wonder why is there a link (`a` tag) for the session name, and without any link destination? I think it would be better to change this maybe using a dimmed color instead, like the syntax name which is displayed in the ST references popup.